lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -293,6 +293,12 @@ def services
       end
     end
 
+    def services_payment_partial_name
+      if @services_data[ceremony_country].present?
+        @services_data[ceremony_country]['payment_partial_name']
+      end
+    end
```

Comment:
I think you could use the (relatively new) Ruby [`Hash#dig`][1] method here to make the code a bit more succinct:

```ruby
def services_payment_partial_name
  @services_data.dig(ceremony_country, 'payment_partial_name')
end
```

[1]: http://ruby-doc.org/core-2.3.0_preview1/Hash.html#method-i-dig